internal/json/types.go

```diff
@@ -418,18 +418,49 @@ func (Encoder) AppendObjectData(dst []byte, o []byte) []byte {
 	return append(dst, o...)
 }
 
-// AppendIPAddr adds IPv4 or IPv6 address to dst.
+// AppendIPAddr adds a net.IP IPv4 or IPv6 address to dst.
```

Comment:
followed the pattern from other types (this was not done when IPAddr was first added)